commit f7d4a3a66f325c7d5731393a9be816c53b96ac77
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2019-04-12T20:02:47+00:00 (6 months ago)
    
    TRITON-1402 Affinity tests failing in cloudapi test mode due to server_uuid
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/machines.test.js b/test/machines.test.js
index a9da6fc6e..3e6496463 100644
--- a/test/machines.test.js
+++ b/test/machines.test.js
@@ -1605,8 +1605,7 @@ test('Delete packageless/nicless machine', deleteMachine);
 test('Affinity tests', function (t) {
     var affinityTest = require('./machines/affinity');
 
-    affinityTest(t, CLIENT, OTHER, IMAGE_UUID, SDC_128.uuid, SERVER_UUID,
-        function () {
+    affinityTest(t, CLIENT, OTHER, IMAGE_UUID, SDC_128.uuid, function affCb() {
         t.end();
     });
 });
diff --git a/test/machines/affinity.js b/test/machines/affinity.js
index 7def7ee04..124ccbb84 100644
--- a/test/machines/affinity.js
+++ b/test/machines/affinity.js
@@ -21,7 +21,7 @@ var CONTAINER_PREFIX = 'sdccloudapitest_affinity_';
 // --- Tests
 
 module.exports =
-function (suite, client, other, imgUuid, pkgUuid, headnodeUuid, cb) {
+function affinityTests(suite, client, other, imgUuid, pkgUuid, cb) {
     var VM_UUID;
     var VM2_UUID;
 
@@ -31,7 +31,6 @@ function (suite, client, other, imgUuid, pkgUuid, headnodeUuid, cb) {
             image: imgUuid,
             package: pkgUuid,
             name: CONTAINER_PREFIX + libuuid.create().split('-')[0],
-            server_uuid: headnodeUuid,
             firewall_enabled: true,
             affinity: [affinity]
         };
